packages/synapse-interface/slices/_transactions/reducer.ts

```diff
@@ -70,6 +72,21 @@ export const transactionsSlice = createSlice({
       )
       if (txIndex !== -1) {
         state.transactions[txIndex].isComplete = true
+        state.transactions[txIndex].status = 'completed'
+      }
+    },
+    revertTransaction: (
+      state,
+      action: PayloadAction<{ originTxHash: string }>
+    ) => {
+      const { originTxHash } = action.payload
+
+      const txIndex = state.transactions.findIndex(
+        (tx) => tx.originTxHash === originTxHash
+      )
+      if (txIndex !== -1) {
+        state.transactions[txIndex].isReverted = true
+        state.transactions[txIndex].status = 'reverted'
```

Comment:
If we have a status here, why do we _also_ need `isReverted`? 

Whole point of `status` is to have one spot which tells us what the status of a given txn is. Same goes for `isComplete` vs status of `completed`

Let's rewrite this flow so we only use `status` and don't have extra redundant variables that make flow hard to follow

Reply:
ah forgot to remove these after adding the `status` prop lol

Reply:
Removed `isCompleted` and `isReverted` props from `_TransactionDetails` type in https://github.com/synapsecns/sanguine/pull/2052/commits/9a9db9eb8d942081e63de250a9494f73c3e7a212

Updated `useBridgeTxUpdater` to check if tx action needs to be dispatched to update `tx.status` in https://github.com/synapsecns/sanguine/pull/2052/commits/edf938e1131e8bb792f2c8bd12c460c5288bd7ed

Removed status variables in favor of explicit status comparisons in https://github.com/synapsecns/sanguine/pull/2052/commits/1148bab81e7152442690ad72329f8979751a1478